presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientConfig.java

```diff
@@ -70,7 +70,7 @@
     private Duration noHostAvailableRetryTimeout = new Duration(1, MINUTES);
     private int speculativeExecutionLimit = 1;
     private Duration speculativeExecutionDelay = new Duration(500, MILLISECONDS);
-    private ProtocolVersion protocolVersion = ProtocolVersion.V3;
```

Comment:
This change isn't in the original PR and isn't currently in Trino? https://github.com/trinodb/trino/commit/6a46ec1551bf20f4580b3af5c89d77e4b83f8e13

Reply:
It's not in original PR, the DATE type in broken for ProtocolVersion <= 3 and I'm trying to fix it.